### PR TITLE
Fix/Сommons

### PR DESCRIPTION
**Pull request by yehor-podporinov**

### CODE REVIEW

File: app.vue [hunk 1]
```diff
@@ -2,8 +2,24 @@
   <NuxtLayout>
```
Comment (ArtSuslov) [inline app.vue:2]: filename should be capitalized

File: layouts/solidity-tools.vue [hunk 2]
```diff
@@ -1,26 +1,47 @@
 <template>
```
Comment (ArtSuslov) [inline layouts/solidity-tools.vue:1]: filename, and in other places

File: pages/abi.vue [hunk 3]
```diff
@@ -1,17 +1,17 @@
 <template>
```
Comment (ArtSuslov) [inline pages/abi.vue:1]: name should contain at least 2 words

File: pages/abi.vue [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
   <div class="abi-page">
```
Comment (ArtSuslov) [inline pages/abi.vue:2]: root class should duplicate the filename

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

File: app.vue (hunk 1 above)
Comment (yehor-podporinov) [reply app.vue:2]: discussed

File: layouts/solidity-tools.vue (hunk 2 above)
Comment (yehor-podporinov) [reply layouts/solidity-tools.vue:1]: discussed

File: pages/abi.vue (hunk 3 above)
Comment (yehor-podporinov) [reply pages/abi.vue:1]: discussed

**Review by ArtSuslov — APPROVED** [review]

File: pages/abi.vue (hunk 4 above)
Comment (yehor-podporinov) [reply pages/abi.vue:2]: discussed

**Pull request merged**